third_party/ibis/ibis_addon/operations.py

```diff
@@ -178,6 +178,9 @@ def strftime_bigquery(translator, op):
     arg_type = arg.output_dtype
     strftime_format_func_name = BQ_STRFTIME_FORMAT_FUNCTIONS[type(arg_type)]
     fmt_string = translator.translate(format_str)
+    # Deal with issue 1181 due a GoogleSQL bug with dates before 1000 CE affects both date and timestamp types
+    if format_str.value.startswith("%Y"):
+        fmt_string = fmt_string.replace("%Y", "%4Y", 1)
```

Comment:
BigQuery documentation states we should use "%E4Y":

https://cloud.google.com/bigquery/docs/reference/standard-sql/format-elements#format_elements_date_time

I see "%4Y" also works but don't see it documented, should we use the documented format? Or did I miss some other documentation?